app/controllers/events_controller.rb

```diff
@@ -94,6 +94,6 @@ def date_from_form(date_info)
 
     # Only allow a trusted parameter "white list" through.
     def event_params
-      params.require(:event).permit(:name, :description, :max_participants, :active, :kind, :organizer, :knowledge_level)
+      params.require(:event).permit(:name, :description, :max_participants, :draft, :kind, :organizer, :knowledge_level)
```

Comment:
Actually, `:draft` is contained directly in `params` since it depends on the button clicked and not on the form content. Therefore, I would leave this out